<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteDialog</class>
 <widget class="QDialog" name="RouteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры маршрута</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Тип транспорта</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="carRadioButton">
          <property name="text">
           <string>Автомобиль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="undergroundRadioButton">
          <property name="text">
           <string>Метро</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="busRadioButton">
          <property name="text">
           <string>Автобус</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="trolleybusRadioButton">
          <property name="text">
           <string>Троллейбус</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tramRadioButton">
          <property name="text">
           <string>Трамвай</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="taxiRadioButton">
          <property name="text">
           <string>Маршрутка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="footRadioButton">
          <property name="text">
           <string>Пешком</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Время передвижения</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Время:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Стоимость</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Стоимость, р.:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="costSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RouteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RouteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
